.travis.yml

```diff
@@ -3,7 +3,4 @@ script:
   - curl https://raw.githubusercontent.com/shapesecurity/CLA/master/cla-travis.sh | bash
   - npm run build
 node_js:
-  - "0.12"
-  - "0.11"
-  - "0.10"
-  - iojs
+  - "5.3"
```

Comment:
The version of npm which ships with 0.12 is apparently too old to build it. I can experiment and find the largest acceptable range, but I don't think it's going to include 0.12.